README.md

```diff
@@ -500,12 +536,12 @@ subscriber.start
 # Shut down the subscriber when ready to stop receiving messages.
 subscriber.stop.wait!
 ```
-### Redis API (Alpha)
+### Cloud Redis API (Alpha)
 
 - [google-cloud-redis README](google-cloud-redis/README.md)
-+- [google-cloud-redis API documentation](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-redis/latest)
+- [google-cloud-redis API documentation](http://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud-redis/latest)
 - [google-cloud-redis on RubyGems](https://rubygems.org/gems/google-cloud-redis)
-- [Redis API documentation](https://cloud.google.com/redis/docs/)
+- [Redis API documentation](https://cloud.google.com/memorystore/docs/redis/)
```

Comment:
@danoscarmike Shouldn't "[Redis API documentation]" be "[Cloud Redis API documentation]"?

Reply:
Great catch. Agreed.

Reply:
@danoscarmike Do you want to make this change or do you want one of us to push a commit to your PR?